File: python/ReverseWords151.py
class Solution1:
	def reverseWords(self, s:str)-> str :
		return ' '.join(s.strip().split()[::-1])

class Solution:
    def reverseWords(self, s: str) -> str:
        return ' '.join(reversed(s.split()))
a = Solution()
v = a.reverseWords("the   sky is ")
print(v)

File: java/ReverseWords151.java
import java.util.Arrays;
import java.util.Collections;

class ReverseWords151 {
	public String reverseWords(String s) {
		String [] words = s.trim().split(" +");
		Collections.reverse(Arrays.asList(words));
		return String.join(" ",words);
	}

	public static void main(String [] args){
		ReverseWords151 s = new ReverseWords151();
		String n = s.reverseWords(" the sky ss    is blue");
		System.out.println(n);
	}
}

